test/unit/specs/avatar.spec.ts

```diff
@@ -0,0 +1,117 @@
+import { mountWithProps, mountWithElement, wait } from '../util'
+import { Avatar } from 'packages/avatar'
+import { IMAGE_SUCCESS, IMAGE_FAIL } from '../mocks/uri'
+
+describe('Avatar', () => {
+  it('create', async () => {
```

Comment:
We should mock native dom that dom is not actually mounted in `jest`.
Here you need to add the code for the mock. 
```
  beforeAll(() => {
    const srcKey = Symbol('src')
    // Mocking Image.prototype.src to call the onload or onerror
    // callbacks depending on the src passed to it
    Object.defineProperty(window.Image.prototype, 'src', {
      // Define the property setter
      set(src) {
        this[srcKey] = src
        if (src === IMAGE_FAIL) {
          // Call with setTimeout to simulate async loading
          setTimeout(() => this.dispatchEvent(new Event('error')))
        } else if (src === IMAGE_SUCCESS) {
          setTimeout(() => this.dispatchEvent(new Event('load')))
        }
      },
      get() {
        return this[srcKey]
      },
    })
  })
```